### PR TITLE
Add badge binding

### PR DESCRIPTION
**Pull request by phthhieu**

### CODE REVIEW

File: examples/Index.re [hunk 1]
```diff
@@ -1,6 +1,18 @@
+[%bs.raw {|require("@ehrocks/eh-ant-design/styles/eh-antd.css")|}];
```
Comment (phamtuan2993) [inline examples/Index.re]: we have already import this to src/Ehd.re, why do we need to re-import it here again a?

Comment (phthhieu) [reply examples/Index.re]: I would remove in the Ehd.re

File: src/Ehd_Badge.re [hunk 2]
```diff
@@ -0,0 +1,59 @@
+module StatusProp = {
+  [@bs.deriving jsConverter]
+  type t = [ | `success | `processing | `default | `error | `warning];
```
Comment (tienle) [inline src/Ehd_Badge.re]: @phthhieu  https://github.com/moroshko/bs-blabla/blob/master/bs.string.md 
Dùng `bs.string` đi, đỡ viết mấy cái wrapper qua lại :D

Comment (phthhieu) [reply src/Ehd_Badge.re]: Coool thanks a. Let me update

**Pull request merged**